build-tools/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "root",
-	"version": "0.15.0",
+	"version": "0.16.0",
```

Comment:
This was missed in the original version bump, likely because of using an earlier version of build-tools to do the bump.